features/step_definitions/run_steps.rb

```diff
@@ -18,10 +18,10 @@
 end
 
 Then(/^warder detects( no)? (.+) (issues|violations)$/) do |no, what, _|
-  executing_output = send(:"executing_#{what.gsub(' ', '_')}")
+  executing_output = send(:"executing_#{what.tr(' ', '_')}")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.